test/performance-charts/tests/PieChart.bench.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+// eslint-disable-next-line no-restricted-imports
+import { render, cleanup } from '@testing-library/react';
+import { bench, describe } from 'vitest';
+import { PieChart } from '@mui/x-charts/PieChart';
+import { options } from '../utils/options';
+
+describe('PieChart', () => {
+  const dataLength = 10;
```

Comment:
Should we have a bit more? Maybe 50 data points? 

Pie doesn't work for thousands, but I suppose we should push it a bit.